auth-microservice/app/tests/conftest.py

```diff
@@ -1,8 +1,8 @@
 import asyncio
-from typing import Dict
-
 import pytest
 
+from typing import Dict
```

Comment:
With python 3.9 you can subscript the built-in `dict`, no need to import it from typing
```suggestion
```

Also replace the `Dict[str, str]` on line 55 with a `dict[str, str`